src/commcare_cloud/ansible/requirements.yml

```diff
@@ -1,6 +1,9 @@
 ---
 - version: v1.1.4
   name: andrewrothstein.couchdb-cluster
+- src: git+https://github.com/andrewrothstein/ansible-couchdb.git
+  version: f0e026e73b8bf6f4ee5cc710f8fdbcdeb7418b2b
+  name: andrewrothstein.couchdb
```

Comment:
Why can't we just `couchdb_ver` instead of pinning to an old version?

Reply:
We probably can! I actually didn't understand what I was doing so much as I was undoing a part of https://github.com/dimagi/commcare-cloud/pull/2215 that looked to me to be suspiciously related.

Is the change you're suggesting basically
- undo this commit (don't include andrewrothstein.couchdb as an explicit dependency here)
- add `couchdb_ver: '2.1.1'` to `all.yml`
? (I got `2.1.1` from the default for that variable in the version pinned in this PR (f0e026e73b8bf6f4ee5cc710f8fdbcdeb7418b2b): https://github.com/andrewrothstein/ansible-couchdb/blob/f0e026e73b8bf6f4ee5cc710f8fdbcdeb7418b2b/defaults/main.yml#L2)